e2e/tests/test_e2e.py

```diff
@@ -858,6 +858,7 @@ def test_multi_namespace_support(self):
         try:
             k8s.create_with_kubectl("manifests/complete-postgres-manifest.yaml")
             k8s.wait_for_pod_start("spilo-role=master", self.test_namespace)
+            k8s.wait_for_pod_start('spilo-role=replica')
```

Comment:
why does waiting or the master uses the `self.test_namespace` and waiting for the replica does not?

Reply:
an oversight, indeed. Might explain why the test fails atm